### PR TITLE
add naga eye close when naga Dazed

### PR DESCRIPTION
**Pull request by ghost**
Naga will closed eyes when Naga hit the shield and confused

### CODE REVIEW

File: src/main/java/twilightforest/client/renderer/entity/RenderTFNaga.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
 public class RenderTFNaga extends RenderLiving<EntityTFNaga> {
 
 	private static final ResourceLocation textureLoc = TwilightForestMod.getModelTexture("nagahead.png");
+	private static final ResourceLocation textureLoc_dazed = TwilightForestMod.getModelTexture("nagahead_dazed.png");
```
Comment (Drullkus) [inline src/main/java/twilightforest/client/renderer/entity/RenderTFNaga.java]: `textureLoc_dazed` should be renamed to `textureLocDazed` and that should do

Comment (Drullkus) [inline src/main/java/twilightforest/client/renderer/entity/RenderTFNaga.java]: `textureLoc_dazed` should be renamed to `textureLocDazed` and that should do

**Review by Drullkus — CHANGES_REQUESTED** [review]

**Comment by Tamaized** [comment]
Wondering if a layer render would be better here. Not sure though.

**Comment by ghost** [comment]
I changed layer :)

File: src/main/java/twilightforest/client/renderer/entity/RenderTFNaga.java [hunk 2]
```diff
@@ -29,6 +30,8 @@ public void doRender(EntityTFNaga entity, double x, double y, double z, float en
 
 	@Override
 	protected ResourceLocation getEntityTexture(EntityTFNaga entity) {
+
 		return textureLoc;
+
```
Comment (Tamaized) [inline src/main/java/twilightforest/client/renderer/entity/RenderTFNaga.java]: 2 stray newlines here but other than that this looks good to me

**Review by Tamaized — APPROVED** [review]

File: src/main/java/twilightforest/client/renderer/entity/NagaEyelidsLayer.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	public void doRenderLayer(T entitylivingbaseIn, float limbSwing, float limbSwingAmount, float partialTicks, float ageInTicks, float netHeadYaw, float headPitch, float scale) {
+		if(entitylivingbaseIn.isDazed()) {
+			this.nagaRenderer.bindTexture(textureLocDazed);
+			this.nagaRenderer.getMainModel().render(entitylivingbaseIn, limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scale);
+		}
+	}
+
+	@Override
+	public boolean shouldCombineTextures() {
+		return false;
```
Comment (Tamaized) [inline src/main/java/twilightforest/client/renderer/entity/NagaEyelidsLayer.java]: Iirc this method tells the layer to turn red when the entity is damaged. Was this intentional to have the eyelids not change to red while the rest of the naga does upon taking damage?

**Pull request merged**